src/arrayMethodIndexOf.js

```diff
@@ -5,8 +5,17 @@
  *
  */
 function applyCustomIndexOf() {
-  [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.indexOf2 = function(searchElement, fromIndex = 0) {
+    const arrayLength = this.length;
+    if (arrayLength === 0 ||
+      fromIndex >= this.length ||
```

Comment:
put the operator at the begin of the line

Reply:
done